Installing-and-Configuring/Upgrading.md

```diff
@@ -157,10 +158,10 @@ sealtotp.sh
 This needs the TPM owner password to be able to define the NVRAM space.
 (todo: [issue #151](https://github.com/osresearch/heads/issues/151)).
 
-Resealing the disk encryption keys
+Resealing the disk encryption keys (optional)
 ---
 
-When you get to the standard boot menu and after you verify the TOTP, select 'm'
+If you want to store the Disk Unlock key in the TPM you will need to do that now.  From the recovery shell run generic-init.  When you get to the standard boot menu and after you verify the TOTP, select 'm'
```

Comment:
"From the recovery shell run generic-init". No.

Same logic applies here. generic-init should not be used for the same reasons that sealing/unsealing secrets cannot be done properly from recovery shell. Please delete. Proper way is from GUI, from other boot options, and selecting a new boot option, accepting the modify disk and use TPM to release the secret (The Disk Unlock Key, released from TPM when proper measurements of LUKS header, firmware, while PCR for boot option is in normal boot (not recovery shell).

Reply:
@tlaurion This is old instructions from the original docs.  It is removed.  Will push in a few mintues.